### PR TITLE
Document recording archive functionality

### PR DESCRIPTION
**Pull request by andrewazores**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,6 +175,17 @@ number and the deployment network configuration allows connections on the
 configured port. As noted above, the final caveat is that in non-Kube
 deployments, port 9091 is expected for automatic port-scanning target discovery.
 
+## ARCHIVING RECORDINGS
+
+`container-jfr` supports a concept of "archiving" recordings. This simply means
+taking a snapshot of a recording at a point in time and saving this snapshot to
```
Comment (jiekang) [inline README.md:181]: Is this different from the existing snapshot recording term? Maybe use a synonym for snapshot?

Comment (andrewazores) [reply README.md]: Reworded it to avoid that potential confusion, since they are different.

File: README.md [hunk 2]
```diff
@@ -175,6 +175,18 @@ number and the deployment network configuration allows connections on the
...
 deployments, port 9091 is expected for automatic port-scanning target discovery.
 
+## ARCHIVING RECORDINGS
+
+`container-jfr` supports a concept of "archiving" recordings. This simply means
+taking the contents of a recording at a point in time and saving these contents
+to a file local to the `container-jfr` process (as opposed to "active"
+recordings, which exist within the memory of the JVM target and continue to grow
+over time). To enable `container-jfr` archive support ensure that the directory
+`/flightrecordings` exists and has appropriate permissions. `container-jfr` will
```
Comment (jiekang) [inline README.md:185]: Should that be `./flightrecordings` ?

Comment (andrewazores) [reply README.md:185]: No, it really is `/flightrecordings`.

**Comment by andrewazores** [comment]
@jiekang did you have any more review comments for this?

**Comment by jiekang** [comment]
@andrewazores Oh my god, sorry I missed this. I don't know why it doesn't show up on my list of PRs to review :(

**Pull request merged**